src/generateZeroFunction.js

```diff
@@ -12,7 +12,10 @@
  *
  */
 function generateZeroFunction() {
-  // write code here
+  const zero = () => {
```

Comment:
Не створюй зайвих змінних. Ти ж можеш зразу повернути значення
```suggestion
 return () => 0
```